diff --git a/build.mill b/build.mill
index 592bbc3..9c193fb 100644
--- a/build.mill
+++ b/build.mill
@@ -6,13 +6,15 @@ import $ivy.`com.github.lolgab::mill-mima::0.1.1`
 import $ivy.`com.goyeau::mill-scalafix::0.5.1`
 import de.tobiasroeser.mill.vcs.version.VcsVersion
 import com.goyeau.mill.scalafix.ScalafixModule
-import mill._, scalalib._, publish._
+import mill._, scalalib._, scalanativelib._, publish._
 
 val scala3 = "3.6.2"
-val scalaVersions = Seq("2.13.12", scala3)
+val scalaVersions = Seq(scala3)
 val scala3NamedTuples = "3.7.0"
 
-trait CommonBase extends ScalaModule with PublishModule with ScalafixModule { common =>
+trait CommonBase extends ScalaNativeModule with PublishModule with ScalafixModule { common =>
+
+  def scalaNativeVersion = "0.5.7"
 
   def publishVersion = VcsVersion.vcsState().format()
 
@@ -82,12 +84,12 @@ object `scalasql-namedtuples` extends CommonBase {
   //   super.scalacOptions() :+ "-Xprint:inlining"
   // }
 
-  object test extends CommonTest {
-    def resources = scalasql(scala3).test.resources
-    def moduleDeps = super.moduleDeps ++ Seq(scalasql(scala3), scalasql(scala3).test)
-    def recordedTestsFile: String = "recordedTestsNT.json"
-    def recordedSuiteDescriptionsFile: String = "recordedSuiteDescriptionsNT.json"
-  }
+  // object test extends CommonTest {
+  //   def resources = scalasql(scala3).test.resources
+  //   def moduleDeps = super.moduleDeps ++ Seq(scalasql(scala3), scalasql(scala3).test)
+  //   def recordedTestsFile: String = "recordedTestsNT.json"
+  //   def recordedSuiteDescriptionsFile: String = "recordedSuiteDescriptionsNT.json"
+  // }
 }
 
 object scalasql extends Cross[ScalaSql](scalaVersions)
@@ -99,19 +101,19 @@ trait ScalaSql extends Common { common =>
 
   override def consoleScalacOptions: T[Seq[String]] = Seq("-Xprint:typer")
 
-  object test extends CommonTest {
-    def recordedTestsFile: String = "recordedTests.json"
-    def recordedSuiteDescriptionsFile: String = "recordedSuiteDescriptions.json"
-  }
+  // object test extends CommonTest {
+  //   def recordedTestsFile: String = "recordedTests.json"
+  //   def recordedSuiteDescriptionsFile: String = "recordedSuiteDescriptions.json"
+  // }
 
   private def indent(code: Iterable[String]): String =
     code.map(_.split("\n").map("  " + _).mkString("\n")).mkString("\n")
 
   object core extends Common with CrossValue {
     def ivyDeps = Agg(
-      ivy"com.lihaoyi::geny:1.0.0",
-      ivy"com.lihaoyi::sourcecode:0.3.1",
-      ivy"com.lihaoyi::pprint:0.8.1"
+      ivy"com.lihaoyi::geny:1.1.1",
+      ivy"com.lihaoyi::sourcecode:0.4.2",
+      ivy"com.lihaoyi::pprint:0.9.0"
     ) ++ Option.when(scalaVersion().startsWith("2."))(
       ivy"org.scala-lang:scala-reflect:${scalaVersion()}"
     )
